math-scala/src/main/scala/org/apache/mahout/drivers/MahoutOptionParser.scala

```diff
@@ -73,40 +75,44 @@ class MahoutOptionParser(programName: String) extends OptionParser[Map[String, A
 
   def parseElementInputSchemaOptions{
     //Input text file schema--not driver specific but input data specific, elements input,
-    // not drms
+    // not rows of IndexedDatasets
     opts = opts ++ MahoutOptionParser.TextDelimitedElementsOptions
     note("\nInput text file schema options:")
-    opt[String]("inDelim") abbr ("id") text ("Input delimiter character (optional). Default: \"[ ,\\t]\"") action { (x, options) =>
-      options + ("inDelim" -> x)
+    opt[String]("inDelim") abbr ("id") text ("Input delimiter character (optional). Default: \"[ ,\\t]\"") action {
+      (x, options) => options + ("inDelim" -> x)
     }
 
     opt[String]("filter1") abbr ("f1") action { (x, options) =>
       options + ("filter1" -> x)
-    } text ("String (or regex) whose presence indicates a datum for the primary item set (optional). Default: no filter, all data is used")
+    } text ("String (or regex) whose presence indicates a datum for the primary item set (optional). " +
+      "Default: no filter, all data is used")
 
     opt[String]("filter2") abbr ("f2") action { (x, options) =>
       options + ("filter2" -> x)
-    } text ("String (or regex) whose presence indicates a datum for the secondary item set (optional). If not present no secondary dataset is collected")
+    } text ("String (or regex) whose presence indicates a datum for the secondary item set (optional). " +
+      "If not present no secondary dataset is collected")
 
     opt[Int]("rowIDColumn") abbr ("rc") action { (x, options) =>
       options + ("rowIDColumn" -> x)
-    } text ("Column number (0 based Int) containing the row ID string (optional). Default: 0") validate { x =>
-      if (x >= 0) success else failure("Option --rowIDColNum must be >= 0")
+    } text ("Column number (0 based Int) containing the row ID string (optional). Default: 0") validate {
+      x => if (x >= 0) success else failure("Option --rowIDColNum must be >= 0")
     }
 
     opt[Int]("itemIDColumn") abbr ("ic") action { (x, options) =>
       options + ("itemIDColumn" -> x)
-    } text ("Column number (0 based Int) containing the item ID string (optional). Default: 1") validate { x =>
-      if (x >= 0) success else failure("Option --itemIDColNum must be >= 0")
+    } text ("Column number (0 based Int) containing the item ID string (optional). Default: 1") validate {
```

Comment:
The larger issue is probably that infix is used and that is because the referenced examples for this code use infix as Scala style dictates. Unless we want to make some decision about that I have no problem with mixing infix and dot styling. These are more of a DSL than a dot method call if that make any difference.